requirements.txt

```diff
@@ -5,3 +5,4 @@ pillow==2.9.0
 psycopg2==2.6.2
 six==1.10.0
 html5lib==1.0b8
+mysql
```

Comment:
Looking at the [package on pypi](https://pypi.org/project/mysql/), we really should just use [`MySQL-python`](https://pypi.org/project/MySQL-python/) instead?  That hasn't been touched since 2014, though; we may want the more up-to-date [`mysqlclient`](https://pypi.org/project/mysqlclient/) instead (which is a fork of `MySQL-python`).

Reply:
@mook-as So a simple change to `mysqlclient` should work without any other code changes ?

Reply:
Change done with commit 3fc443f26350fceb5375b263b73cb1c27e0ab237